src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/SARIF1003.UrisMustBeValid_Valid.sarif

```diff
@@ -28,7 +28,7 @@
         }
       },
       "originalUriBaseIds": {
-        "SRCROOT": "file:///c:/Code/sarif-sdk/src"
+        "SRCROOT": "file:///c:/Code/sarif-sdk/src/"
```

Comment:
>/ [](start = 49, length = 1)

This file is supposed to be valid against SARIF1003 ("URIs must be valid"). Don't let it be invalid because of the new SARIF1019.